### PR TITLE
[Reference cleanup]LPS-161060

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in analytics/*. Thanks!
https://issues.liferay.com/browse/LPS-161060

previous PR:https://github.com/liferay-core-infra/liferay-portal/pull/1401

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:analytics-cloud

File: modules/apps/analytics/analytics-settings-web/src/main/java/com/liferay/analytics/settings/web/internal/portal/settings/configuration/admin/display/BaseAnalyticsConfigurationScreen.java [hunk 1]
```diff
@@ -100,19 +100,17 @@ protected String getJspPath() {
...
 	@Reference
 	protected AnalyticsUsersManager analyticsUsersManager;
 
 	@Reference
 	protected ConfigurationProvider configurationProvider;
 
+	@Reference(
+		target = "(&(release.bundle.symbolic.name=com.liferay.analytics.settings.web)(release.schema.version>=1.0.1))"
+	)
+	protected Release release;
```
Comment (dantewang) [inline modules/apps/analytics/analytics-settings-web/src/main/java/com/liferay/analytics/settings/web/internal/portal/settings/configuration/admin/display/BaseAnalyticsConfigurationScreen.java:112]: @JanisZhang Does this need to be kept as protected to ensure sub-classes can inherit the DS annotation? Or it's related to something else?

Comment (JanisZhang) [reply modules/apps/analytics/analytics-settings-web/src/main/java/com/liferay/analytics/settings/web/internal/portal/settings/configuration/admin/display/BaseAnalyticsConfigurationScreen.java:112]: Hi @dantewang, based on the failure (https://github.com/liferay-core-infra/liferay-portal/pull/1401#issuecomment-1220381214)
I saw this module has `-dsannotations-options: inherit` by default, it seems like the subclasses want to access the parent class field.  so I think this need to be kept as protected to ensure sub-classes can inherit the DS annotation 
```
!ENTRY com.liferay.analytics.settings.web 4 0 2022-08-19 07:30:25.034

!MESSAGE bundle com.liferay.analytics.settings.web:5.0.32 (908)[com.liferay.analytics.settings.web.internal.portal.settings.configuration.admin.display.AnalyticsSyncedContactsDataConfigurationScreen(8613)] : findField: Suitable but non-accessible field _release in class com.liferay.analytics.settings.web.internal.portal.settings.configuration.admin.display.BaseAnalyticsConfigurationScreen, subclass of component class com.liferay.analytics.settings.web.internal.portal.settings.configuration.admin.display.AnalyticsSyncedContactsDataConfigurationScreen


!ENTRY com.liferay.analytics.settings.web 4 0 2022-08-19 07:30:25.038

!MESSAGE bundle com.liferay.analytics.settings.web:5.0.32 (908)[com.liferay.analytics.settings.web.internal.portal.settings.configuration.admin.display.AnalyticsSyncedSitesConfigurationScreen(8614)] : findField: Suitable but non-accessible field _release in class com.liferay.analytics.settings.web.internal.portal.settings.configuration.admin.display.BaseAnalyticsConfigurationScreen, subclass of component class com....
```

Comment (dantewang) [reply modules/apps/analytics/analytics-settings-web/src/main/java/com/liferay/analytics/settings/web/internal/portal/settings/configuration/admin/display/BaseAnalyticsConfigurationScreen.java:112]: @JanisZhang Then could you check whether the `@Reference Release` in `AnalyticsMessageSenderClientImpl` can be private? This class itself is a component and it doesn't have subclass.

**Pull request closed without merging**